src/ApiPort/ApiPort.project.json

```diff
@@ -8,8 +8,8 @@
     "Microsoft.Framework.Configuration.CommandLine": "1.0.0-beta5",
     "Microsoft.Framework.OptionsModel": "1.0.0-beta5",
     "Newtonsoft.Json": "6.0.8",
-    "System.Collections.Immutable": "1.1.37",
-    "System.Reflection.Metadata": "1.1.0-alpha-00008",
+    "System.Collections.Immutable": "1.1.38-*",
+    "System.Reflection.Metadata": "1.2.0-alpha-00017",
```

Comment:
Yup - it was a corner case causing a failure on methods with an arglist parameter. It should be fixed in alpha-17 - are you seeing any tests failing? There was one test failing, so if all tests are passing, it's fixed.